bad.js

```diff
@@ -0,0 +1,4 @@
+var a = function(){
```

Comment:
'a' is assigned a value but never used.
Missing space before function parentheses.
Missing space before opening brace.